code/datums/components/pellet_cloud.dm

```diff
@@ -102,7 +114,7 @@
   *
   * Note that grenades have extra handling for someone throwing themselves/being thrown on top of it, while landmines do not (obviously, it's a landmine!). See [/datum/component/pellet_cloud/proc/handle_martyrs()]
   */
-/datum/component/pellet_cloud/proc/create_blast_pellets(mob/living/lanced_by=NONE)
+/datum/component/pellet_cloud/proc/create_blast_pellets(obj/O, mob/living/lanced_by=NONE)
```

Comment:
```suggestion
/datum/component/pellet_cloud/proc/create_blast_pellets(obj/O, mob/living/lanced_by)
```
Either this or `= null`
`NONE` is a define for bitflags.

Reply:
done